### PR TITLE
Add/update checkpoint notices on guides

### PR DESCRIPTION
**Pull request by fhildeb**
As the checkpoint synchronization is currently down, this puts a maintenance note on the guide pages where people can find further information. (linked to the CLI README)

-> Related PR: (CLI README) [Updated Checkpoint Commands](https://github.com/lukso-network/tools-lukso-cli/pull/220)

### CODE REVIEW

**Review by Hugoo — APPROVED** [review]

File: docs/networks/advanced-guides/switch-consensus-clients.md [hunk 1]
```diff
@@ -171,6 +165,12 @@ lukso start --checkpoint-sync
 lukso start --validator --transaction-fee-recipient "0x1234" --checkpoint-sync
 ```
 
+:::caution
+
+If you are experiencing issues with the `--checkpoint-sync` flag or are using a LUKSO CLI Version below version 0.7.0, the checkpoint synchronization can be [enabled manually](https://github.com/lukso-network/tools-lukso-cli/tree/main#checkpoints-with-lukso-cli-version-07-or-below).
```
Comment (CJ42) [inline docs/networks/advanced-guides/switch-consensus-clients.md:170]: minor, but repeating the word version 2 times

```suggestion
If you are experiencing issues with the `--checkpoint-sync` flag or are using a LUKSO CLI version below 0.7.0, the checkpoint synchronization can be [enabled manually](https://github.com/lukso-network/tools-lukso-cli/tree/main#checkpoints-with-lukso-cli-version-07-or-below).
```

File: docs/networks/advanced-guides/update-clients.md [hunk 2]
```diff
@@ -99,6 +99,12 @@ lukso start --checkpoint-sync
 lukso start --validator --transaction-fee-recipient "0x1234" --checkpoint-sync
 ```
 
+:::caution
+
+If you are experiencing issues with the `--checkpoint-sync` flag or are using a LUKSO CLI Version below version 0.7.0, the checkpoint synchronization can be [enabled manually](https://github.com/lukso-network/tools-lukso-cli/tree/main#checkpoints-with-lukso-cli-version-07-or-below).
```
Comment (CJ42) [inline docs/networks/advanced-guides/update-clients.md:104]: same

File: docs/networks/mainnet/become-a-validator.md [hunk 3]
```diff
@@ -107,9 +107,9 @@ If you are setting up a node for the testnet, add the `--testnet` flag to the st
 
 :::
 
-:::note
+:::caution
 
-You will need the LUKSO CLI Version 0.8.0 or above in order to use the `--checkpoint-sync` command. If you are using an older version, please pass down the checkpoint flag as described in the [LUKSO CLI Documentation](https://github.com/lukso-network/tools-lukso-cli/tree/main#using-checkpoint-syncing).
```
Comment (CJ42) [inline docs/networks/mainnet/become-a-validator.md:112]: Shouldn't this still be kept as a mention that the `checkpoint-sync` command is available only from v0.8.0?

File: docs/networks/mainnet/running-a-node.md [hunk 4]
```diff
@@ -104,20 +104,7 @@ If you want more convenience and your validator to operate quickly, you can also
 
 > After the synchronization is finalized, you will end up with the equal blockchain data. You can use the flag on every startup. However, it shows the most significant effect when synchronizing from scratch or after an extended downtime. The shortcut is ideal for fresh installations, validator migration, or recovery.
 
-<Tabs>
```
Comment (CJ42) [inline docs/networks/mainnet/running-a-node.md:107]: Why removing this? Once the checkpoint sync is not down anymore, this section will still be useful to have no?

Comment (fhildeb) [reply docs/networks/mainnet/running-a-node.md:107]: its not removed- the order of the menu bar changed (its added down below). checkpoint sync should stay as first option

**Comment by fhildeb** [comment]
This PR is closed due to #702

**Pull request closed without merging**